src/PrestaShopBundle/Resources/views/Admin/macros.html.twig

```diff
@@ -92,18 +92,83 @@
 {% macro form_widget_with_error(form, vars, extraVars) %}
   {% set vars = vars|default({}) %}
   {% set extraVars = extraVars|default({}) %}
+  {% set attr = vars.attr|default({}) %}
+  {% set errors = [] %}
+
+  {% if not form.vars.valid  %}
```

Comment:
👍

Reply:
Can you explain a bit the possible usecases in twig comments for `form_widget_with_error` macro ?

Twig macros are powerful but also hard to decypher 😉 let's help people reuse them

And later we can make it a devdocs section